test1/CellViewController.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  CellViewController.swift
+//  test1
+//
+//  Created by Nikola Blagoevski on 22.4.22.
+//
+
+import UIKit
+import SnapKit
+import SDWebImage
+
+class NikolaAwesomeLabel: UILabel{
+    override init(frame: CGRect){
+        super.init(frame: frame)
+        self.backgroundColor = .white
+        self.font = .systemFont(ofSize: 20)
+        self.textAlignment = .left
+        self.numberOfLines = 0
+        self.lineBreakMode = .byWordWrapping
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+class CellViewController: UIViewController{
+    
+
+    var person: CharacterModel
+    //let tableView = UITableView()
+    
+    init(person: CharacterModel){
+        self.person = person
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        //view.addSubview(tableView)
+//      tableView.delegate = self
+//      tableView.dataSource = self
+        makeStackView()
+    }
+    
+    lazy var image: UIImageView = {
+        let temp = UIImageView()
+        temp.sd_setImage(with: URL(string: person.image))
+        return temp
+    }()
+    
+    lazy var label1: NikolaAwesomeLabel = {
```

Comment:
we need better names than label1.2.3 etc

Reply:
the names of the labels are changed and are easy to understand